<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eci2kep_test</class>
 <widget class="QWidget" name="eci2kep_test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="TestCount">
         <property name="text">
          <string>#### Tests</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Inspect Error:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Overall</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Period</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>a</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>e</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>h</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>beta angle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>eta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>i</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>raan</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ap</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>alat</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ma</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ea</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mm</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>fa</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="TestList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="testsOverview">
       <property name="text">
        <string>Max error in #### Min error in ####</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="SelectMax">
         <property name="text">
          <string>Select Max Error</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SelectMin">
         <property name="text">
          <string>Select Min Error</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Test #### - (section)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QWidget" name="testInfo" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
